src/CMakeLists.txt

```diff
@@ -423,6 +423,7 @@ add_files(
     stringfilter.cpp
     stringfilter_type.h
     strings.cpp
+    strings_internal.h
```

Comment:
The alphabet says: no! :D

Reply:
https://en.wikipedia.org/wiki/I_before_E_except_after_C :D